### ISSUE TITLE
series: change return type of 'slices' method

### ISSUE THREAD

**Issue by FridaTveit** [exercism/java#1598]
Currently the `slices` method in the [series exercise](https://github.com/exercism/java/tree/master/exercises/series) returns a list of string where each string represents a digit. Instead it should return a list of strings where each string is a number. That way it would match the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/series/canonical-data.json).

**Comment by FridaTveit** [comment]
I believe @hgvanpariya is working on this. Let me know if I'm wrong @hgvanpariya 🙂

**Comment by AndresAcuna** [comment]
I understand @hgvanpariya is working on this, I was just trying to understand the request.

The return type of `slices` should be `List<String>` and not `List<List<String>>`?

**Comment by hgvanpariya** [comment]
@FridaTveit : yes , I'm working on this....

**Comment by FridaTveit** [comment]
Yes, that's right @AndresAcuna!  Sorry, I didn't explain it very well 🙂
